diff --git a/platform_specific.mk.intel-mac b/platform_specific.mk.intel-mac_now
index b260f4f..a056421 100644
--- a/platform_specific.mk.intel-mac
+++ b/platform_specific.mk.intel-mac_now
@@ -103,9 +103,7 @@ endif
 
 	CFLAGS		= -fPIC $(MYDEFINES) $(MYINCLUDES)
 
-	FFLAGS		= -fPIC \
-			  -fallow-invalid-boz \
-			  -fallow-argument-mismatch \
+	FFLAGS		= -fPIC -static-libgfortran \
 			  -fno-automatic \
 			  -fno-second-underscore \
 			  -fdollar-ok \
