diff --git a/platform_specific.mk.intel-mac b/platform_specific.mk.intel-mac-new
index b260f4f..55b6b24 100644
--- a/platform_specific.mk.intel-mac
+++ b/platform_specific.mk.intel-mac-new
@@ -104,6 +104,7 @@ endif
 	CFLAGS		= -fPIC $(MYDEFINES) $(MYINCLUDES)
 
 	FFLAGS		= -fPIC \
+                          -static-libgfortran \
 			  -fallow-invalid-boz \
 			  -fallow-argument-mismatch \
 			  -fno-automatic \
@@ -118,6 +119,7 @@ endif
 			  $(MYDEFINES) $(MYINCLUDES)
 
 	PPLUS_FFLAGS	= -fPIC \
+                          -static-libgfortran \
 			  -fno-automatic \
 			  -fallow-invalid-boz \
 			  -fallow-argument-mismatch \
