--- external_functions/ef_utility/platform_specific.mk.x86_64-linux
+++ external_functions/ef_utility/platform_specific.mk.x86_64-linux
@@ -22,8 +22,10 @@
 LD		= gfortran
 LD_DYN_FLAGS	= -fPIC -m64 -shared -Wl,--no-undefined
-SYSLIBS		= ../ef_utility/copy_ferret_ef_mem_subsc.o \
-		  -L$(INSTALL_FER_DIR)/lib64/$(PYTHON_EXE)/site-packages/pyferret \
-		  -L$(INSTALL_FER_DIR)/lib/$(PYTHON_EXE)/site-packages/pyferret \
-		  -lpyferret
+# Some compilers add an OS spec between libpyferret and .so
+# This also takes care of whether to use lib or lib64
+LIBPYFERRET	= $(wildcard $(SP_DIR)/pyferret/libpyferret*.so)
+LIBPYFERRET_DIR	= $(shell dirname $(LIBPYFERRET))
+LIBPYFERRET_NAM	= $(shell basename $(LIBPYFERRET) .so | sed -e 's/libpyferret/pyferret/' )
+SYSLIBS		= ../ef_utility/copy_ferret_ef_mem_subsc.o -L$(LIBPYFERRET_DIR) -l$(LIBPYFERRET_NAM)
 
 # Directly compile the .F source files to the .o object files
