--- external_functions/ef_utility/platform_specific.mk.x86_64-linux	2016-08-26 21:50:00.000000000 -0300
+++ external_functions/ef_utility/platform_specific.mk.x86_64-linux	2016-09-07 11:20:01.299770866 -0300
@@ -21,11 +21,8 @@
 FFLAGS_DEBUG	= -O0 -g -fbounds-check 
 
 LD		= gfortran
-LD_DYN_FLAGS	= -fPIC -m64 -shared -Wl,--no-undefined
-SYSLIBS		= ../ef_utility/copy_ferret_ef_mem_subsc.o \
-		  -L$(INSTALL_FER_DIR)/lib64/$(PYTHON_EXE)/site-packages/pyferret \
-		  -L$(INSTALL_FER_DIR)/lib/$(PYTHON_EXE)/site-packages/pyferret \
-		  -lpyferret
+LD_DYN_FLAGS	= -fPIC -bundle -undefined dynamic_lookup
+SYSLIBS		= ../ef_utility/copy_ferret_ef_mem_subsc.o 
 
 # Directly compile the .F source files to the .o object files
 # since gfortran can handle the C compiler directives in Fortran code
